### ISSUE TITLE
Problem with reading shortcuts configs. There are no shortcuts found in preferences. 

### ISSUE THREAD

**Issue by nortmas** [Ulauncher/Ulauncher#169]
**Steps to reproduce:**
Open Ulauncher preferences, shortcuts section. There are no shortcuts found. However, in settings I see many shortcuts:

```
{
    "baa7fe3f-172b-4e11-89ef-b7c175f1e892": {
        "added": 1506942037.534021, 
        "name": "Stack Overflow", 
        "keyword": "so", 
        "cmd": "http://stackoverflow.com/search?q=%s", 
        "is_default_search": true, 
        "id": "baa7fe3f-172b-4e11-89ef-b7c175f1e892", 
        "icon": "/usr/share/ulauncher/media/stackoverflow-icon.svg"
    }, 
    "64961746-44b7-492d-80b5-b5ea2832192c": {
        "added": 1506942037.534091, 
        "name": "Wikipedia", 
        "keyword": "wiki", 
        "cmd": "https://en.wikipedia.org/wiki/%s", 
        "is_default_search": true, 
        "id": "64961746-44b7-492d-80b5-b5ea2832192c", 
        "icon": "/usr/share/ulauncher/media/wikipedia-icon.png"
    }, 
    "abbaba1a-ba7f-4264-8c89-b075003d2296": {
        "added": 1507000735.309825, 
        "name": "Drupal Modules", 
        "keyword": "dm", 
        "cmd": "https://www.drupal.org/search/site/%s?f%5B0%5D=ss_meta_type%3Amodule", 
        "is_default_search": false, 
        "id": "abbaba1a-ba7f-4264-8c89-b075003d2296", 
        "icon": "~/pictures/dr.png"
    }, 
    "2bf52990-35a4-4b73-80b3-40467f8d787c": {
        "added": 1506942037.533946, 
        "name": "Google Search", 
        "keyword": "g", 
        "cmd": "https://google.com/search?q=%s", 
        "is_default_search": true, 
        "id": "2bf52990-35a4-4b73-80b3-40467f8d787c", 
        "icon": "/usr/share/ulauncher/media/google-search-icon.png"
    }
}
```
 

**Ulauncher version**: v3.2.0r1
**Linux distro**: Ubuntu 14.04
**Desktop environment**: Unity

**Logs**:

[https://gist.github.com/nortmas/541bb40cca3efc0819d1b527c1cea45b](https://gist.github.com/nortmas/541bb40cca3efc0819d1b527c1cea45b)

**Comment by gornostal** [comment]
Interesting. Did you copy the JSON above from `~/.config/ulauncher/shortcuts.json`?

**Comment by nortmas** [comment]
Yep, exactly form that place.
BTW I have to laptops. One has Ubuntu 14.04 and another Ubuntu 16.04.
For Ubuntu 16.04 everything works perfectly.

**Comment by gornostal** [comment]
Ah, I see the problem. Will fix it soon

**Comment by gornostal** [comment]
Fixed in v3.2.1
